tests/suite/s3/parquet.yaml

```
@@ -0,0 +1,14 @@
+script: |
+  source minio.sh
+  mv *.tzng ./data/bucket
+  zq -i parquet dns.parquet
+
+inputs:
+  - name: minio.sh
+    source: ../zar/s3/minio.sh
+  - name: dns.parquet
+    source: ./testdata/dns.parquet
+
+outputs:
+  - name: stdout
+    source: ./testdata/dns.zng
```

Nit: No good reason to put this in testdata.
```suggestion
    source: dns.zng
```